static/bidder-info/stroeerCore.yaml

```diff
@@ -0,0 +1,16 @@
+maintainer:
+  email: "help@cz.stroeer-labs.com"
+gvlVendorID: 136
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+  site:
+    mediaTypes:
+      - banner
+userSync:
+  iframe:
+    url: "https://js.adscale.de/pbsync.html?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&redirect={{.RedirectURL}}"
+  redirect:
+    url: "https://ih.adscale.de/uu?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&cburl={{.RedirectURL}}"
+  default: "iframe"
```

Comment:
Please remove the `default` line. A recent change removed this field and hardcoded iframe as the default.